components/glew/Makefile

```diff
@@ -0,0 +1,100 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov <jimklimov@gmail.com>
+#
+
+# NOTE: This project links with /usr/openwin/lib/libGL.so.1, which unrolls
```

Comment:
If we do this, we should do this globally for components, linking to libGL.so. But it seems we have not been beaten by the issue yet. I'd preserved thins as they are for now - added 
depend fmri=service/opengl/ogl-select type=require and  <transform file -> add pkg.depend.bypass-generate libGL\.so\.1>